overrides.go

```diff
@@ -0,0 +1,72 @@
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at http://mozilla.org/MPL/2.0/.
+//
+// Contributor:
+// - Brandon Myers bmyers@mozilla.com
+
+package main
+
+import (
+  "fmt"
+  "os"
+  "bufio"
+  "strings"
+  "strconv"
+)
+
+type Override struct {
+  CIDR string
+  City string
+  Country string
+  Latitude float64
+  Longitude float64
+}
+
+
+func readOverrides(path string) (overrides []Override){
+  file, err := os.Open(path)
+  defer file.Close()
+  if err != nil {
+    fmt.Printf("error opening file: %v\n",err)
+    return overrides
+  }
+
+  reader := bufio.NewReader(file)
+
+  var line string
+  for {
+    line, err = reader.ReadString('\n')
```

Comment:
You might want to use bufio.Scanner here

https://golang.org/pkg/bufio/#Scanner

Reply:
Nice, I thought that code was pretty hacky when I wrote it. I implemented that in https://github.com/ameihm0912/geomodel/pull/10/commits/7924535a214c3750e5ab7e2b4d0e4058e63dc2c1 and it's much, much cleaner.